dev/Repeater/FlexboxLayoutState.h

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+class FlexboxLayoutState :
+    public winrt::implements<FlexboxLayoutState, winrt::IInspectable>
+{
+public:
+    struct RowMeasureInfo
+    {
+    public:
+        float MainAxis;
+        float CrossAxis;
+        int Count;
+        float Grow;
+        float Shrink;
+        float Basis;
```

Comment:
initialize these as values are not zero-initialized in C++.
Name them according to the guidelines (m_ for members)
```suggestion
        float m_mainAxis{};
        float m_crossAxis{};
        int m_count{};
        float m_grow{};
        float m_shrink{};
        float m_basis{};
```